src/metabase/lib/filter.cljc

```diff
@@ -241,14 +244,14 @@
              (operator-def :is-null :is-empty)
              (operator-def :not-null :not-empty)])]
     ;; The order of these clauses is important since we want to match the most relevant type
-    (condp #(lib.types.isa/isa? %2 %1) column
```

Comment:
`lib.types.isa/isa?` is not a faithful port of the corresponding JS function and it was causing problems with coordinates.
I'm not sure where this function is used, if it's important, then we should revise how it should work.